src/go/Makefile

```diff
@@ -136,15 +142,16 @@ darwin-amd64:				## Build Mongo tools for darwin-amd64 (MacOS)
 
 style:						## Check code style
 	@echo ">> checking code style"
-	@! gofmt -d $(shell find . -path ./vendor -prune -o -name '*.go' -print) | grep '^'
+	@! bin/gofumpt -d $(shell find . -path ./vendor -prune -o -name '*.go' -print) | grep '^'
 
 test:						## Run tests
 	@echo ">> running tests"
 	@./runtests.sh
 
 format:                         ## Format source code.
-	gofumpt -w -s $(FILES)
-	gofumports -local github.com/percona/pmm-managed -l -w $(FILES)
+	go mod tidy
+	bin/gofumpt -w -s $(FILES)
+	bin/gofumports -local github.com/percona/pmm-managed -l -w $(FILES)
```

Comment:
You can leave only `gofumports` because it's equal to `gofumt` + impors sorting

Reply:
Actually `gofumports` was deprecated. So you need `gofumpt + goimports`
https://github.com/percona/pmm/blob/9450d3d000fa7a35fbbdd12511688e4c5bff1165/Makefile.include#L163-L164